addons/mail/static/src/models/notification_group/notification_group.js

```diff
@@ -10,20 +10,13 @@ registerModel({
     identifyingFields: ['res_model', 'res_id', 'notification_type'],
     recordMethods: {
         /**
-         * Opens the view that allows to cancel all notifications of the group.
+         * Discard notifications of the group.
          */
-        openCancelAction() {
-            if (this.notification_type !== 'email') {
-                return;
-            }
-            this.env.bus.trigger('do-action', {
-                action: 'mail.mail_resend_cancel_action',
-                options: {
-                    additional_context: {
-                        default_model: this.res_model,
-                        unread_counter: this.notifications.length,
-                    },
-                },
+        notifyCancel() {
```

Comment:
I don't like that we use 2 terms for the same thing (discard and cancel), we should pick one and stick to it.